File: app/Models/Recovery.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;

class Recovery extends Model 
{

    /**
     * The attributes that are mass assignable.
     *
     * @var array
     */
    protected $fillable = [
        'email', 
        'ip', 
        'domain',
        'code', 
        'access_id'
    ];


}

File: cmd/app/models/recovery.go
package models

import "time"

type Recovery struct {
	ID        uint   `gorm:"primary_key"`
	Email     string `gorm:"type:varchar(100);not null"`
	IP        string `gorm:"type:varchar(100);not null"`
	Domain    string `gorm:"type:varchar(100);not null"`
	Code      string `gorm:"type:varchar(100);not null"`
	AccessID  uint   `gorm:"not null"`
	CreatedAt time.Time
	UpdatedAt time.Time
}
